### PR TITLE
For Issue (Create BranchRepositoryService)

### PR DESCRIPTION
**Pull request by ziyadullayevmirjalol**

### CODE REVIEW

File: CodeHub.Service/Services/BranchRepositoryService.cs [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+        this.mapper = mapper;
+        this.Repository = Repository;
+        this.userService = userService;
+        this.repositoryService = repositoryService;
+    }
+
+    public async Task<BranchRepositoryViewModel> CreateAsync(BranchRepositoryCreateModel branchRepository)
+    {
+        var existUser = await userService.GetByIdAsync(branchRepository.UserId);
+        var existRepository = await repositoryService.GetByIdAsync(branchRepository.UserId);
```
Comment (makhammadsoliyev) [inline CodeHub.Service/Services/BranchRepositoryService.cs]: You should use branchRepository.RepositoryId

Comment (Asrorbek-Abrorov) [reply CodeHub.Service/Services/BranchRepositoryService.cs]: +

Comment (ziyadullayevmirjalol) [reply CodeHub.Service/Services/BranchRepositoryService.cs]: ok

**Review by makhammadsoliyev — APPROVED** [review]
Good

**Pull request merged**